test/contracts/watchable-function-logic/StorageDuplicate.sol

```diff
@@ -1,26 +0,0 @@
-// SPDX-License-Identifier: MIT
```

Comment:
This was being used for the test: `should throw error if contract name is ambiguous`. Why deleting it?